sunpy/time/astropy_time.py

```diff
@@ -0,0 +1,91 @@
+import astropy.time
+
+import numpy as np
+
+from time import strftime, strptime
+from datetime import date, datetime
+
+
+class Time(astropy.time.Time):
```

Comment:
When this is merged into astropy we should attempt to use the one in astropy if it exists. i.e. 

```python
if hasattr(astropy.time.Time, strptime):
    from astropy.time import TIme
else:
    class Time(...
```